spec/factories/pacbio/pool.rb

```diff
@@ -7,7 +7,7 @@
       library_factory { :pacbio_library }
     end
 
-    primary_aliquot { association :aliquot, source: instance, aliquot_type: :primary }
+    primary_aliquot { association :aliquot, source: instance, aliquot_type: :primary, volume: 10 }
```

Comment:
You can use the volume variable passed in so that the primary aliquot and pool have the same volume. You can update this to just `volume:` which is shorthand for `volume: volume`.